### DISCUSSION TITLE
When to use dash vs x?

### DISCUSSION THREAD

**Discussion by lespaul4** [iSoron/uhabits#1615] [Help & FAQ]
I'm wondering that is the difference between using a dash (-) versus using the X. I'm assuming X means it was not completed, while the dash means ?

Thanks,

**Comment by rancidfrog** [comment]
I had the same question, which is why I suggested having a legend to explain the differences:
https://github.com/iSoron/uhabits/discussions/1326

 to clarify the icons for skip day and missed habit, as there is nowhere showing which is which

One represents skipped, maybe the dash
The other missed, probably the X

Normally when you skip it should not affect the score, but I am not sure if this is the case on this app

**Comment by hiqua** [comment]
Added "What do the dashes means?" in https://github.com/iSoron/uhabits/discussions/689.
